lib/identity-provider.js

```diff
@@ -526,10 +526,17 @@ IdentityProvider.prototype.getGraph = function (uri, callback) {
 
 // Handle POST requests on account creation
 IdentityProvider.prototype.post = function (req, res, next) {
+  const idpMode = req.app.locals.ldp.idp
```

Comment:
I'm going to push back on this. That line seems completely legit to me.  (I also don't think the IDP mode should be stored in `ldp` anyways. So if anything, it should be req.app.locals.idp)